README.md

```diff
@@ -2,20 +2,23 @@ Bacardi
 =======
 
 [![Linux/Mac Build Status](https://travis-ci.org/lunchclass/bacardi.svg?branch=master)](https://travis-ci.org/lunchclass/bacardi)
+[![Build status](https://ci.appveyor.com/api/projects/status/vk8qn0pilfg39x0a/branch/master?svg=true)](https://ci.appveyor.com/project/romandev/bacardi/branch/master)
 
 Bacardi project is an effort to provide multi-language binding for Node.js
 native layer.
 
 ## System requirements
-- This project is multi-platform oriented, but we support Linux and Mac for now.
-  - We have a plan to support for MS Windows platform but you can try it using
-    Docker for now.
+- This project is multi-platform oriented, so we support Linux, Mac, Windows for now.
+
 #### Linux
 - You MUST have git, python, wget and g++ compilers.
   - ```sudo apt-get install g++ git make python wget```
 #### Mac
 - You MUST have Xcode with command line tools. You can install them as follows:
   - ```xcode-select --install```
+#### Windows
+- You MUST have git. and others will be installed automatically when bacardi command executed.
```

Comment:
You MUST have git. Others will be installed automatically when ```bacardi``` command is executed.